archivebox/parsers/__init__.py

```diff
@@ -42,25 +42,29 @@
 from .generic_json import parse_generic_json_export
 from .generic_html import parse_generic_html_export
 from .generic_txt import parse_generic_txt_export
+from .url_list import parse_url_list
 
-PARSERS = (
+PARSERS = {
     # Specialized parsers
-    ('Pocket API', parse_pocket_api_export),
-    ('Wallabag ATOM', parse_wallabag_atom_export),
-    ('Pocket HTML', parse_pocket_html_export),
-    ('Pinboard RSS', parse_pinboard_rss_export),
-    ('Shaarli RSS', parse_shaarli_rss_export),
-    ('Medium RSS', parse_medium_rss_export),
+    'pocket-api': ('Pocket API', parse_pocket_api_export),
```

Comment:
minor change, can you use the underscore versions of these names to match the filenames (e.g. `pocket_api` instead of `pocket-api`). One less unique string to keep track of if these match the filenames exactly.